### DISCUSSION TITLE
How to add a hamburger menu to the theme and center the actual menu?

### DISCUSSION THREAD

**Discussion by RenaudILTIS** [datenstrom/community#922] [Ask a question] [answered]
Hello hello,

Could you help me with this question ? At least some guidelines, I can do my homework 😉
Only for the small screens (for example < 600px) and with no submenus.

Thank you very much.

**Comment by RenaudILTIS** [comment]
I just succeed install the "miniburger" plugin from @GiovanniSalmeri, but there is an issue : 
![hamburger](https://github.com/datenstrom/yellow/assets/5601919/1bbe7d26-25e5-439c-aea4-0d0405e887b7)

The menu is not showing the current page.

Do you have an idea of the problem/solution ?

**Comment by RenaudILTIS** [comment, marked as answer]
OK, I changed
`@media screen and (max-width: 600px) {
  .topnav a:not(:first-child) {display: none;}`
to
 `@media screen and (max-width: 600px) {
  .topnav a {display: none;}`
In `miniburger.css` and it's OK now.

**Comment by annaesvensson** [comment]
Tip: You can let us know when your question has been answered, click on `Mark as answer`.

**Reply by RenaudILTIS** [reply]
Hello Anna, thank you very much for all your good work !
(By the way, the link to the Helsinki template is broken).

**Reply by annaesvensson** [reply]
Thank you Renaud for your kind feedback. Did you mean Helsinki theme? I don't think there is a Helsinki theme for Datenstrom Yellow, but it's a beautiful city and maybe someone should make one. 😆 Available extensions can be found here: https://datenstrom.se/yellow/extensions/

**Reply by schulle4u** [reply]
Indeed we once had such a theme, at least it was mentioned in the release announcement for version 0.8.3 (#412). Maybe someone still has it and can adapt the theme for the current version?

**Reply by schulle4u** [reply]
Seems it is still available. After researching the archives we found out that the former Knowhow/Helsinki theme is now known as [Copenhagen](https://github.com/annaesvensson/yellow-copenhagen).

**Reply by RenaudILTIS** [reply]
> I don't think there is a Helsinki theme for Datenstrom Yellow, but it's a beautiful city and maybe someone should make one. 😆 

I researched were I saw it and in fact it's not on the datenstrom website 😄, it's here : [https://in.10437.lima.zone/themes/](https://in.10437.lima.zone/themes/)
Regarding the links, the previsous location of the theme was [https://github.com/datenstrom/yellow-extensions/tree/master/themes/helsinki](https://github.com/datenstrom/yellow-extensions/tree/master/themes/helsinki)

But as @schulle4u mentionned, it seems very similar to Copenhagen.
